### PR TITLE
add effect layer to asset container

### PR DESCRIPTION
**Pull request by TrevorDev**
https://github.com/BabylonJS/Babylon.js/issues/5324

### CODE REVIEW

File: src/babylon.assetContainer.ts [hunk 1]
```diff
@@ -65,10 +65,9 @@ module BABYLON {
             this.textures.forEach((o) => {
                 this.scene.addTexture(o);
             });
-
-            for (let component of this.scene._serializableComponents) {
-                component.addFromContainer(this.scene);
```
Comment (TrevorDev) [inline src/babylon.assetContainer.ts]: @sebavan it looks like you added this section that I removed. It looks like component.addFromContainer(this.scene); has logic specific to effect layer and keeping it had unexpected results. Is it needed now that I added effectLayer?

Comment (sebavan) [reply src/babylon.assetContainer.ts]: Effect Layer should only be in components not directly in the container. It has been moved during the module work. 

What is the issue after the move ?

Comment (TrevorDev) [reply src/babylon.assetContainer.ts]: ok it looks like there was a bug with this.scene when it should be this. Also I still want to be abloe to do asset container.effectLayers.push(layer) is there a way you recomend that?

Comment (sebavan) [reply src/babylon.assetContainer.ts]: Is the problem the empty list on creation ?

Comment (TrevorDev) [reply src/babylon.assetContainer.ts]: should be good now as we discussed

**Pull request merged**